addons/account/views/account_invoice_view.xml

```diff
@@ -424,7 +428,11 @@
                 <div groups="account.group_account_invoice" class="alert alert-info" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('has_outstanding','=',False)]}">
                     You have <bold><a class="alert-link" href="#outstanding" role="button">outstanding payments</a></bold> for this customer. You can allocate them to mark this invoice as paid.
                 </div>
+                <div groups="account.group_account_invoice" class="alert alert-info" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('edition_mode_available','=',False)]}">
+                    You have suspense account moves that match this invoice. <bold><button class="alert-link" type="object" name="action_reconcile_to_check" role="button" string="Check them" style="padding: 0 0 1px 0;"/></bold> to mark this invoice as paid.
```

Comment:
@csnauwaert can you check that button? The style is weird and I've tried to change with `<a action="action_reconcile_to_check">Check them</a>`, the display is good then but it's not clickable anymore... -_-

Reply:
@qdp-odoo did you try adding a type= object to the a tag? <a type="object" action="action_reconcile_to_check">Check them</a>
It should works with the type and the action.

Reply:
@csnauwaert It doesn't work